editor/credits_roll.cpp

```diff
@@ -0,0 +1,235 @@
+/*                        https://godotengine.org                         */
+/**************************************************************************/
+/* Copyright (c) 2014-present Godot Engine contributors (see AUTHORS.md). */
+/* Copyright (c) 2007-2014 Juan Linietsky, Ariel Manzur.                  */
+/*                                                                        */
+/* Permission is hereby granted, free of charge, to any person obtaining  */
+/* a copy of this software and associated documentation files (the        */
+/* "Software"), to deal in the Software without restriction, including    */
+/* without limitation the rights to use, copy, modify, merge, publish,    */
+/* distribute, sublicense, and/or sell copies of the Software, and to     */
+/* permit persons to whom the Software is furnished to do so, subject to  */
+/* the following conditions:                                              */
+/*                                                                        */
+/* The above copyright notice and this permission notice shall be         */
+/* included in all copies or substantial portions of the Software.        */
+/*                                                                        */
+/* THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,        */
+/* EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF     */
+/* MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. */
+/* IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY   */
+/* CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,   */
+/* TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE      */
+/* SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.                 */
+/**************************************************************************/
+
+#include "credits_roll.h"
+
+#include "core/authors.gen.h"
+#include "core/donors.gen.h"
+#include "core/input/input.h"
+#include "core/license.gen.h"
+#include "core/string/string_builder.h"
+#include "editor/themes/editor_scale.h"
+#include "editor_string_names.h"
+#include "scene/gui/box_container.h"
+#include "scene/gui/color_rect.h"
+#include "scene/gui/label.h"
+#include "scene/gui/texture_rect.h"
+
+Label *CreditsRoll::_create_label(const String &p_with_text, LabelSize p_size) {
+	Label *label = memnew(Label);
+	label->set_horizontal_alignment(HORIZONTAL_ALIGNMENT_CENTER);
+	label->set_h_size_flags(Control::SIZE_SHRINK_CENTER);
+	label->set_text(p_with_text);
+
+	switch (p_size) {
+		case LabelSize::NORMAL:
+			label->add_theme_font_size_override(SNAME("font_size"), font_size_normal);
+			break;
+
+		case LabelSize::HEADER:
+			label->add_theme_font_size_override(SNAME("font_size"), font_size_header);
+			break;
+
+		case LabelSize::BIG_HEADER:
+			label->add_theme_font_size_override(SNAME("font_size"), font_size_big_header);
+			break;
```

Comment:
Headers and big headers could use the bold font for better readability, like we do elsewhere in the editor.